### DISCUSSION TITLE
Daily 15/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#58] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer solo pude participar de la clase de aplicaciones móviles.
Hoy voy a realizar el trabajo de testing y voy a avanzar con el de aplicaciones móviles.

**Comment by BraianTroncoso** [comment]
Ayer estuve hasta tarde viendo el tema con el mvp y no tuve problemas para la conexion, hoy voy a ver si puedo hacer el crud y despues si tengo tiempo vincularlo con el views. También mas tarde tengo clases de Ingles.

**Comment by MarielaGomez** [comment]
Ayer tuve clases de inglés y toma de evidencia oral, intento ponerme a día con los nuevos contenidos que van agregando, se dificulta llevar al día.
